### ISSUE TITLE
installer wont let me use /Applications/Eclipse.app

### ISSUE THREAD

**Issue by maxandersen** [eclipse-platform/.github#58]
![2022-08-31_09-44-55](https://user-images.githubusercontent.com/54129/187623146-2d4d2df7-e447-4ea1-ad5b-4c52efc5b747.png)

not sure why i'm allowed to edit the install folder but it wont let me install ?

**Comment by merks** [comment]
I expect that the installer process does not have write access to the /Applications folder.  If you do run it as "su" then you'd better be careful and read all the details on this page:

https://www.eclipse.org/setups/installer/question/

I.e., be careful where the bundle pool is located or disable that.

**Comment by maxandersen** [comment]
@merks it does have write access. Since just putting /Applications it talks about /Applications/Eclipse.app already existing and then running install works.

This does not have much to do with `su` but with misleading UI message here.

**Comment by maxandersen** [comment]
What i'm after is having *one* folder where I have eclipse installed. Yes, I could put this in ~/eclipse/Eclipse.app too - but that gives same error.

**Comment by merks** [comment]
Please see this link for reporting problems:

https://www.eclipse.org/setups/installer/problem/

But note that I only have a mac virtual box so fixing Mac things is a challenge at best.

The application will necessarily be called Eclipse.app  (just as that is the name in the *.dmg) so you should not specify that part of the name but rather you should specify a folder that can be created in which Eclipse.app will in turn will exist.  This won't change...

**Comment by maxandersen** [comment]
May I suggest that the UI message actually tells why folder named Eclipse.app is not accepted? 

The behaviour of this dialog is just hard to follow when it comes to warnings/errors.

Now when I run installer I get this:

![2022-08-31_14-36-34](https://user-images.githubusercontent.com/54129/187679784-48543a99-5bee-4bf1-a35c-ff69c770fa0f.png)


if I put "/Users/max/" it tells me "The folder is specified as /Users/max/ but /Users/max will be used' - those are equivalent.

If I put "/Users/max/eclipse" when "Eclipse.app" exist I get a warning "Product folder '/Users/max/eclipse/Eclipse.app/Contents/Eclipse" already exists. What is "Product" folder ? it is not used in the other warning/errors.

If I click Install it actually installs on top of existing install.

And now the kicker - if I put "/Users/max/eclipse/Eclipse.app" then it does not behave the same as when "/Applications/Eclipse.app" is specified. No warning/error given.

**Comment by merks** [comment]
You may suggest many things, but this is not the place to report a problem nor to make suggestions about the installer.  It's hard to even comment on the state of your file system.  I don't even know if you've already installed an application inside the another application's folder and in which cases you've done that or not.  

It would of course be best if you tried to contribute your suggestions because somehow macos is always a bit 'special':

https://www.eclipse.org/setups/installer/?url=http://git.eclipse.org/c/oomph/org.eclipse.oomph.git/plain/setups/configurations/OomphConfiguration.setup&show=true

**Comment by maxandersen** [comment]
I gave up trying to find where to report issues - this repo was what bugs.eclipse.org points to when trying to open issues and not knowing where to put.

afaics there is nothing osx special here though - similar happens on Windows.

Which is the right place to open issues on this?

**Comment by merks** [comment]
I did mentioned where to look in this response:

https://github.com/eclipse-platform/.github/issues/58#issuecomment-1232838434

Here's the direct link:

https://bugs.eclipse.org/bugs/enter_bug.cgi?product=Oomph&component=Setup&short_desc=Installer
